docs/usage/quickstart.rst

```diff
@@ -3,85 +3,119 @@
    Quickstart
    subdirectory
    Todo
-   usernames
 
 Quickstart Guide
 ================
-To quickly set up a testing cluster using MetalK8s_, you need 3 machines running
-CentOS_ 7.4 to which you have SSH access (these can be VMs). Each machine
-acting as a Kubernetes_ node (all of them, in this example) also need to have at
-least one disk available to provision storage volumes.
+
+To set up a testing cluster quickly using MetalK8s_ requires three machines
+running CentOS_ 7.4 to which you have SSH access (these can be VMs). Each
+machine acting as a Kubernetes_ node (all of them, in this example) must also
+have at least one disk available to provision storage volumes.
 
 .. todo:: Give some sizing examples
```

Comment:
this is still a TODO

Reply:
I'm working on that, can remain as-is.

Reply:
If this will be user-facing (on the doc website), I would suggest that instead of having a 'todo', we can say something like: "Sizing examples will be added when available in the future", or something similar.

Reply:
Agree, though there are better ways to handle that using Sphinx :smiley:

Anyway, out of scope for this PR.